hack/update-bazel.sh

```diff
@@ -43,9 +42,11 @@ if ! grep -q "# gazelle:importmap_prefix" "${KUBE_ROOT}/vendor/BUILD"; then
   echo "# gazelle:importmap_prefix k8s.io/kubernetes/vendor" >> "${KUBE_ROOT}/vendor/BUILD"
 fi
 
-gazelle fix \
-    -build_file_name=BUILD,BUILD.bazel \
-    -external=vendored \
-    -mode=fix
+pushd "$(kube::realpath ${KUBE_ROOT})" > /dev/null 2>&1
```

Comment:
Made this slightly less awful by using `kube::realpath`

Reply:
I have a slight preference on subshell instead of pushd/popd, i.e.

```bash
(
  cd wherever
  gazelle ...
)
```